### ISSUE TITLE
Crashes SDK support for Unity UWP

### ISSUE THREAD

**Issue by pavel-shchelkun-ilogos** [microsoft/appcenter#806]
At the moment, AppCenter Crashes SDK 2.2.0 does not support Unity UWP platform. It would really help to have this feature.

As an alternative, Microsoft Partner Center provides crash report tracking through Analytics Health section. But it does not help a lot, as it usually registers crashes as 'unknown' due to storage or privacy constrains. In some occasions, it does register crashes with system API calls, but it is not that helpful in comparison to Unity script calls stack-traces.

For exceptions, we can also use Unity Cloud Diagnostics, but it is also restricted and does not support crashes on UWP due to some platform limitation related to crash report interception.

In addition to that, it would be great to have not only crashes supported in the SDK, but also exceptions, errors and even custom logs tracking with AppCenter Diagnostics.

**Comment by winnieli1208** [comment]
Hi @PavloShchelkuniLogos - SDK release 2.1.0 supports Unity UWP. Let me know if that works for you!

**Comment by PavloShchelkuniLogos** [comment]
Hi @winnieli1208 last week we tested SDK release 2.2.0 and crashes module did not work. Analytics module is working fine, but we need crashes module. According to Surjyo from support team of App Center whom I contacted after we finished testing 2.2.0, he said that Unity UWP was still not supported for crashes module.

Here is support matrix from docs:
https://docs.microsoft.com/en-us/appcenter/general/platform-service-matrix
